### PR TITLE
refactor: enum renaming

### PR DESCRIPTION
**Pull request by wfxr**
Make enums more readable.

### CODE REVIEW

File: internal/database/database.go [hunk 1]
```diff
@@ -71,15 +71,15 @@ func CreateMetadataDatabase(ctx context.Context, opt types.MetadataStoreConfig)
...
+	case types.BackendMySQL:
 		return offlineMySQL.Open(opt.MySQL)
-	case types.SNOWFLAKE:
+	case types.BackendSnowFlake:
 		return offlineSnowflake.Open(opt.Snowflake)
-	case types.BIGQUERY:
+	case types.BackendBigQuery:
 		return offlineBigQuery.Open(ctx, opt.BigQuery)
-	case types.REDSHIFT:
+	case types.BackendRedShift:
```
Comment (YikSanChan) [inline internal/database/database.go:82]: BackendRedshift

File: internal/database/database.go [hunk 2]
```diff
@@ -71,15 +71,15 @@ func CreateMetadataDatabase(ctx context.Context, opt types.MetadataStoreConfig)
...
 func OpenOfflineStore(ctx context.Context, opt types.OfflineStoreConfig) (offline.Store, error) {
 	switch opt.Backend {
-	case types.POSTGRES:
+	case types.BackendPostgres:
 		return offlinePG.Open(opt.Postgres)
-	case types.MYSQL:
+	case types.BackendMySQL:
 		return offlineMySQL.Open(opt.MySQL)
-	case types.SNOWFLAKE:
+	case types.BackendSnowFlake:
```
Comment (YikSanChan) [inline internal/database/database.go:78]: BackendSnowflake

File: internal/database/database.go [hunk 3]
```diff
@@ -28,15 +28,15 @@ import (
...
+	case types.BackendPostgres:
 		return onlinePG.Open(opt.Postgres)
-	case types.REDIS:
+	case types.BackendRedis:
 		return onlineRedis.Open(opt.Redis), nil
-	case types.MYSQL:
+	case types.BackendMySQL:
 		return onlineMySQL.Open(opt.MySQL)
-	case types.DYNAMODB:
+	case types.BackendDynamodb:
```
Comment (YikSanChan) [inline internal/database/database.go:37]: BackendDynamoDB

**Review by YikSanChan — APPROVED** [review]
lgtm

**Comment by wfxr** [comment]
> lgtm. Let's use their official names in our code:
> 
> RedShift -> Redshift
> SnowFlake -> Snowflake
> Dynamodb -> DynamoDB

@YikSanChan updated

**Review by lianxmfor — APPROVED** [review]
lgtm

**Pull request merged**